app/models/manageiq/providers/dummy_pro/inventory/collector/cloud_manager.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::DummyPro::Inventory::Collector::CloudManager < ManagerRefresh::Inventory::Collector
+  def connection
+    @connection ||= manager.connect
+  end
+
+  def vms
```

Comment:
can you add a comment here about what this should do?

Reply:
yeah - definitely